ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -4,6 +4,41 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function createIntallJunit() {
+  if test -f "${SHARED_DIR}/install-status.txt"
+  then
+    EXIT_CODE=`cat ${SHARED_DIR}/install-status.txt | awk '{print $1}'`
+    INSTALL_STATUS=`cat ${SHARED_DIR}/install-status.txt | awk '{print $2}'`
+    if [ "$EXIT_CODE" ==  0  ];then
+      cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="install" tests="2" failures="0">
+        <testcase name="install test"/>
+        <testcase name="bootstrap test"/>
+      </testsuite>
+EOF
+    elif [ $INSTALL_STATUS == "bootstrap" ] || [ $INSTALL_STATUS == "cluster_created" ]
```

Comment:
if the install_status says "cluster_created" why does that mean that "install test" failed?

Reply:
Normally it should not. But if there are other activities in the script after this, it might fail the overall step. In that case, the message will help us narrow down to code after the cluster creation command.